src/Yubikey/Validate.php

```diff
@@ -269,7 +269,9 @@ public function check($otp, $multi = false)
 
         $clientId = $this->getClientId();
         if ($clientId === null) {
-            throw new \InvalidArgumentException('Client ID cannot be null');
+            if (!$this->getUseSecure())
+                throw new \InvalidArgumentException('Client ID cannot be null');
+            $clientId = 1;
```

Comment:
I think refactoring things a bit here to just not add the `clientId` value to the `$params` array might be a better way to handle this than hard-coding an ID like this.